osx/KBKit/KBKit/Component/KBHelperTool.m

```diff
@@ -96,6 +116,7 @@ - (void)refreshComponent:(KBRefreshComponentCompletion)completion {
       if ([bundleVersion isGreaterThan:runningVersion]) {
         if (bundleVersion) info[@"Bundle Version"] = [bundleVersion description];
         self.componentStatus = [KBComponentStatus componentStatusWithInstallStatus:KBRInstallStatusInstalled installAction:KBRInstallActionUpgrade info:info error:nil];
+        [self doInstallAlert:bundleVersion runningVersion:runningVersion];
```

Comment:
Is this correct? `doInstallAlert: bundleVersion`?

Reply:
It compiles and runs correctly. What's the concern?  You're wondering if there should be a space after the `:`?  I guess with objc, the first param can go unnamed and directly after the function name, but the other params need there names.  Or something.  Just following the other calls in the rest of the code.

Reply:
I was just confused about what was being assigned: `doInstallAlert` and `bundleVersion` don't seem to have much to do with one another.

Reply:
Unless it just means "do the install with an alert"

Reply:
It's a function call, calling `doInstallAlert` with 2 params, `bundleVersion` and `runningVersion`.  The function pops up an alert about the newly installed version, depending on what the two params are.

Reply:
Oh yeah. God it's so hard to read with some params being named and some not being named.